pkg/redirectpolicy/manager.go

```diff
@@ -439,6 +450,12 @@ func (rpm *Manager) deletePolicyService(svcID k8s.ServiceID) {
 		config := rpm.policyConfigs[rp]
 		for _, m := range config.frontendMappings {
 			rpm.deletePolicyFrontend(config, m.feAddr)
+			// Retores the svc backends if there's still such a k8s svc.
+			swg := lock.NewStoppableWaitGroup()
+			restored := rpm.svcCache.EnsureService(svcID, swg)
+			if restored {
+				log.Info("Restored service ", svcID)
```

Comment:
Please use `log.WithFields()` to specify the `svcID`

Reply:
Done.